src/Orchard.Web/Modules/Orchard.Comments/Services/CommentService.cs

```diff
@@ -60,6 +60,11 @@ IMessageService messageService
         public Localizer T { get; set; } 
         public ILogger Logger { get; set; }
 
+        public IContentQuery<CommentPart, CommentPartRecord> GetCommentsForContainer(int id) {
```

Comment:
Remove this method and call ` _commentService.GetComments().Where(c => c.CommentedOnContainer == id)` directly in the driver.

Adding a method in the interface is a breaking change, which can't be shipped in 1.10.x branch.